src/components/buttons/join-us.js

```diff
@@ -24,13 +25,13 @@ module.exports = {
               })
               const prospectembed = new EmbedBuilder()
               .setTitle(`Prospect Notification`)
-              .setDescription(`<@${interaction.user.id}> just picked up the <@&${PROSPECT}> role!`)
+              .setDescription(`${member.displayName} just picked up the <@&${PROSPECT}> role!`)
```

Comment:
Instead of completely removing the @ mentions can we just include the display name after the @ mention. so like `<@${interaction.user.id}> (${member.displayName})` or whatever looks nice. it is nice to have the @ mention still there because you can click on it and message them, see what servers they are in, ect